Product/Production/Services/PatientCorrelationCore/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/proxy/PatientCorrelationProxyWebServiceSecuredImpl.java

```diff
@@ -178,5 +183,36 @@ public AddPatientCorrelationResponseType addPatientCorrelation(final PRPAIN20130
 
         LOG.debug("End addPatientCorrelation");
         return response;
+
+    }
+
+    @Override
+    public void addPatientCorrelationPLQ(PatientLocationQueryResponseType plqRecords,
+        AssertionType assertion) {
+        AddPatientCorrelationPLQSecuredRequestType request = new AddPatientCorrelationPLQSecuredRequestType();
+        request.setPatientLocationQueryResponse(plqRecords);
+
+        try {
+            final String url = oProxyHelper
+                .getUrlLocalHomeCommunity(NhincConstants.PATIENT_CORRELATION_SECURED_SERVICE_NAME);
+
+            if (plqRecords == null) {
+                LOG.error(NULL_ERROR);
+            } else {
+
+                final ServicePortDescriptor<PatientCorrelationSecuredPortType> portDescriptor = new PatientCorrelationSecuredAddPLQServicePortDescriptor();
+
+                final CONNECTClient<PatientCorrelationSecuredPortType> client = CONNECTClientFactory.getInstance()
+                    .getCONNECTClientSecured(portDescriptor, url, assertion);
+
+                client.invokePort(PatientCorrelationSecuredPortType.class,"addPatientCorrelationPLQ", request);
+            }
+        }
+        catch (final Exception ex) {
```

Comment:
Dont catch generic exception.

Reply:
Remaining consistent with the rest of the application.

Reply:
Remaining consistent with the rest of the application.